content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+Sourcegraph 4.0, the latest version of our code intelligence platform, is now available. It includes more than 12 new features and enhancements to help you spend more time in flow and less time manually grokking your codebase. 
+
+Our code intelligence platform semantically indexes and analyzes code and its underlying structure to help you derive knowledge from the codebase. It provides devs and engineering leaders with a complete understanding of the codebase to make faster and better decisions. Our platform enables anyone who creates or interacts with code to save time by helping you understand, fix, and automate changes across your entire codebase.
+
+Read on to discover more details about the improvement with Sourcegraph 4.0.  
+
+### 🌅 Code intelligence: more useful, more times daily for every dev
+
+
+<Badge text="Code Search" color="cerise" size="small" onClick={() => window.open("/code-search", "_self")} />
```

Comment:
Using an `onClick` JavaScript handler is not accessible to screen reader users and prevents things like opening in a new tab. It should be a semantic link (`<a>`).

I think this should work but if not @sourcegraph/content-platform should be able to help:
```suggestion
<a href="/code-search"><Badge text="Code Search" color="cerise" size="small" /></a>
```

Same for the other badges.